.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,4 +9,9 @@
 
 * [ ] This PR adds new unconventional dependencies following the process described [here](../CONTRIBUTING.md/#adding-new-unconventional-dependencies)
 
+### New Features
+
+* [ ] Have you added the new feature to an [integration test](../INTEGRATION_TESTS.md)?
+	* [ ] Did you use `cdk-integ` to deploy the infrastructure and generate the snapshot (i.e. `cdk-integ --no-dry-run`)?
```

Comment:
We don't have to pass `--no-dry-run`, just not passing the flag is good enough.

Reply:
Updated.